treeherder/log_parser/artifactbuilders.pyx

```diff
@@ -86,7 +96,7 @@ class BuildbotLogViewArtifactBuilder(ArtifactBuilderBase):
             HeaderParser(),
             StepParser(check_errors=check_errors)
         ]
-        self.name = "Structured Log"
+        self.name = "text_log_summary"
```

Comment:
I've just thought - we'll need to (a) change the UI to reference this new artifact name (https://github.com/mozilla/treeherder-ui/search?q=Structured+Log) and (b) add a temporary conversion in the service, so queries look for both names when trying to retrieve the artefacts.